resources/common/common.tf

```diff
@@ -148,6 +148,7 @@ data "terraform_remote_state" "etcd" {
         bucket = "${var.remote_state_bucket}"
         key = "etcd.tfstate"
         region = "${var.remote_state_region}"
+        role_arn = "${var.aws_account["role_arn"]}"
```

Comment:
@llinder It looks like, you can set empty string for role_arn empty block for assume_role - at least it works for my regular IAM user (no assuming role case):
```
# Generated by scripts/gen-provider.sh
terraform {
  required_version = "0.9.3"
  backend "s3" {
    bucket = "077108534746-kubev1-terraform"
    key    = "vpc.tfstate"
    region = "us-west-2"
    encrypt = "true"
    role_arn = ""
  }
}
```

And:
```

provider "aws" {
  profile = "uit-itlab"
  max_retries = "10"
  region = "us-west-2"
  assume_role {
    role_arn = ""
  }
  allowed_account_ids = [ "000000000000" ]
}
```
My new scripts/provider.sh is in test branch. Do you  have a setup that can test with AWS_ROLE_NAME defined? Thanks!